### PR TITLE
Show filtering criteria

### PR DESCRIPTION
**Pull request by jleblond**

### CODE REVIEW

File: src/components/filters/FiltersInfos.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  filters,
+  description,
+}) => {
+  const appliedFilters = Object.keys(filters).filter(key => filters[key]?.value)
+
+  const getValue = (key) => {
+    const filterValue = filters[key].value
+
+    switch(description[key].type) {
+      case 'SELECT':
```
Comment (romgrk) [inline src/components/filters/FiltersInfos.js:14]: Use the constants, not literally `'SELECT'` or `'RANGE'`.

File: src/components/filters/FiltersInfos.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+          return option.label
+        })
+        return labels.join(', ')
+        break;
+      case 'RANGE':
+        if (filterValue.min || filterValue.max)
+          return `${filterValue.min} to ${filterValue.max} `
+        else
+          throw new Error('MIN and MAX values not defined for Filter range')
+        break;
```
Comment (romgrk) [inline src/components/filters/FiltersInfos.js:27]: `else` and `break` not required, `return` and `throw` branches catch all cases. `break` on line 21 is also superfluous..

File: src/components/filters/FiltersInfos.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        const valuesArray = [].concat(filterValue)
+        const labels = valuesArray.map((val) => {
+          const option = description[key].options.find(option => option.value === val)
+          return option.label
+        })
+        return labels.join(', ')
+        break;
+      case 'RANGE':
+        if (filterValue.min || filterValue.max)
+          return `${filterValue.min} to ${filterValue.max} `
```
Comment (romgrk) [inline src/components/filters/FiltersInfos.js:24]: Won't this show eg `undefined to 10` if only `max` is defined? Also your `if` condition checks for falsy values, but `0` is a falsy value that we probably want to show.

File: src/components/filters/FiltersInfos.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        return [].concat(filterValue).join(', ')
+    }
+  }
+
+  return (
+    <>
+      {appliedFilters.map(key => (
+        <div>
+          <strong>{description[key].label}: </strong>
+          { getValue(key) }
```
Comment (romgrk) [inline src/components/filters/FiltersInfos.js:38]: Inconsistent spaces inside braces.

**Review by UlysseFG — APPROVED** [review]

File: src/components/filters/FiltersInfos.js [hunk 5]
```diff
@@ -9,24 +10,24 @@ const FiltersInfos = ({
...
-      case 'RANGE':
-        if (filterValue.min || filterValue.max)
-          return `${filterValue.min} to ${filterValue.max} `
-        else
-          throw new Error('MIN and MAX values not defined for Filter range')
-        break;
+      case FILTER_TYPE.RANGE:
+        let value = ""
+        if (filterValue.min !== undefined) {value += ` min: ${filterValue.min}`}
+        if (filterValue.max !== undefined) {value +=` max: ${filterValue.max}`}
```
Comment (romgrk) [inline src/components/filters/FiltersInfos.js:25]: Lint:
 - Missing space after equal.
 - If are using braces for a single statement, but other single-statement ifs dont use braces (such as the one right below)

Also the space before ` min` and ` max` isn't ideal. I would at least put it after so it's not visible.
Also I would have implemented it such that only `min = 42` would render as `more than 42`, `max = 42` would render as `less than 42`, and `min = 0, max = 10` would render as `from 0 to 10` or `between 0 and 10`.

**Pull request merged**